web/src/styles.css

```diff
@@ -146,7 +146,7 @@ body {
     -moz-box-shadow: 0 0 5px 1px rgba(0,0,0,0.3);
 }
 #header #header-content {
-    display: block;
+    display: flex;
```

Comment:
Is that required? Changing the display type will require us to test on old devices (including my old iPad) because some bugs might arise..

Reply:
It makes it much easier to center the text properly when resizing. According to https://caniuse.com/#search=flex it would only be a potential issue in IE, and I've made sure it works correctly at least in IE11. If we're worried about support for older versions of IE then it might be a problem. What version is your ipad? I could also try to test on mine.